ngmix/tests/test_runners.py

```diff
@@ -0,0 +1,179 @@
+import pytest
```

Comment:
I have the same comment here as I did on the guessers. We should add tests to make sure things are reproducible and change when we change the seeds.